UpdatedImageProcessing/__init__.py

```diff
@@ -0,0 +1,5 @@
+from Classifiers import *
+from settings import ImgProcSettings
+from logger import Logger
+
+print("shit")
```

Comment:
Please remove this

Reply:
Removed in https://github.com/FlintHill/SUAS-Competition/pull/97/commits/5b99f5dba88c8915e7270ad090163a92629686c1